cubitpy/mesh_creation_functions.py

```diff
@@ -194,11 +208,20 @@ def extrude_mesh_normal_to_surface(
                 # Check that the normal and position are equal.
                 other_coordinates = node_id_pos_normal_map[node_id][0]
                 other_normal = node_id_pos_normal_map[node_id][1]
-                if (
-                    np.linalg.norm(my_coordinates - other_coordinates) > 1e-10
```

Comment:
Why do we need to loosen the tolerances here?

Reply:
Not necessarily. I restored them.